hs_geographic_feature_resource/page_processors.py

```diff
@@ -103,6 +103,7 @@ def landing_page(request, page):
         context['ori_coverage_form'] = ori_coverage_form
         context['geom_information_form'] = geom_information_form
 
+    context['edit_mode'] = edit_resource
```

Comment:
@pkdash no, user cannot delete a file in View mode. The deletion icon only shows up in Edit mode, and when user clicks that deletion icon a default warning popup will show up for any res type, in feature res type we hacked this default popup, replacing it with our customized popup that tells user "delete any of the mandatory shapefile will remove all shapefiles".  To do this, we need to let the frontend know what mode it is in: View or Edit.